apps/antalmanac/src/components/Calendar/AsyncCalendarCard.tsx

```diff
@@ -0,0 +1,155 @@
+import { Collapse, IconButton, Alert, AlertTitle, Box, Typography, useTheme } from '@mui/material';
+import { ExpandMore, InfoOutlined } from '@mui/icons-material';
+import { useEffect, useMemo, useState } from 'react';
+import AppStore from '$stores/AppStore';
+import { getLocalStorageTempSaveData } from '$lib/localStorage';
+import { setTempSaveData } from '$stores/localTempSaveDataHelpers';
+import { useIsMobile } from '$hooks/useIsMobile';
+
+interface TbaSection {
+  courseTitle: string;
+  sectionCode: string;
+}
+
+export default function AsyncCalendarCard() {
+  const isMobile = useIsMobile();
+  if (isMobile) return null;
+
+  const [updateTrigger, setUpdateTrigger] = useState(0);
+  const [collapsed, setCollapsed] = useState(false);
+  const [visible, setVisible] = useState(false);
+
+  const scheduleIndex = AppStore.getCurrentScheduleIndex();
+  const theme = useTheme();
+
+  useEffect(() => {
+    const handleUpdate = () => setUpdateTrigger((prev) => prev + 1);
+    AppStore.on('addedCoursesChange', handleUpdate);
+    AppStore.on('removedCoursesChange', handleUpdate);
+    AppStore.on('clearSchedule', handleUpdate);
+    AppStore.on('currentScheduleIndexChange', handleUpdate);
+    return () => {
+      AppStore.off('addedCoursesChange', handleUpdate);
+      AppStore.off('removedCoursesChange', handleUpdate);
+      AppStore.off('clearSchedule', handleUpdate);
+      AppStore.off('currentScheduleIndexChange', handleUpdate);
+    };
+  }, []);
+
+  useEffect(() => {
+    const handleScreenshot = () => {
+      if (collapsed && visible) {
+        setCollapsed(false);
+      }
+    };
+    AppStore.on('screenshot', handleScreenshot);
+    return () => {
+      AppStore.off('screenshot', handleScreenshot);
+    };
+  }, [collapsed, visible]);
+
+  useEffect(() => {
+    const raw = getLocalStorageTempSaveData();
+    if (!raw) return;
```

Comment:
nit: always use braces
```suggestion
    if (!raw) {
    	return;
    }
```